src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/OverseasEntitiesService.java

```diff
@@ -80,6 +76,38 @@ public ResponseEntity<Object> createOverseasEntity(Transaction transaction,
         return ResponseEntity.created(URI.create(submissionUri)).body(overseasEntitySubmissionCreatedResponseDto);
     }
 
+    public ResponseEntity<Object> updateOverseasEntity(Transaction transaction,
+                                                       String submissionId,
+                                                       OverseasEntitySubmissionDto overseasEntitySubmissionDto,
+                                                       String requestId,
+                                                       String userId) {
+        ApiLogger.debugContext(requestId, "Called updateOverseasEntity(...)");
+
+        if (!hasExistingOverseasEntitySubmission(transaction)) {
+            return ResponseEntity.badRequest().body(String.format(
+                    "Transaction id: %s does not have an existing Overseas Entity submission", transaction.getId()));
+        }
+
+        // TODO Check Mongo for existence of an Overseas Entity linked to the supplied transaction...
+
+        var overseasEntitySubmissionDao = overseasEntityDtoDaoMapper.dtoToDao(overseasEntitySubmissionDto);
+
+        overseasEntitySubmissionDao.setId(submissionId);
+        overseasEntitySubmissionDao.setHttpRequestId(requestId);
+
+        // TODO Do we need to record the 'modified on' date and maintain the 'created on' date? Modified By user should in theory be the same...
+
+        overseasEntitySubmissionsRepository.save(overseasEntitySubmissionDao);
```

Comment:
do we need to save here if the call to  updateOverseasEntitySubmissonWithMetaData on next line will save as well